newIDE/app/src/Export/Builds/index.js

```diff
@@ -3,12 +3,15 @@ import React, { Component } from 'react';
 import { type AuthenticatedUser } from '../../Profile/AuthenticatedUserContext';
 import BuildsList from './BuildsList';
 import { getBuilds, type Build } from '../../Utils/GDevelopServices/Build';
+import { type Game } from '../../Utils/GDevelopServices/Game';
 import BuildsWatcher from './BuildsWatcher';
 
 type Props = {|
   onBuildsUpdated?: () => void,
   authenticatedUser: AuthenticatedUser,
-  gameId: string,
+  game?: ?Game,
+  onGameUpdated?: Game => void,
+  isGameLoading: boolean,
```

Comment:
Nitpicking: can't you consider game === null as loading?

Reply:
No because you could open the builds dialog before you game is registered, and then game will be null.
Or we can decide to hide the button to see the builds, if the game is not registered & loaded